src/storage/src/controller.proto

```diff
@@ -18,7 +18,7 @@ message ProtoCollectionMetadata {
     string consensus_uri = 2;
     string data_shard = 3;
     string remap_shard = 4;
-    string status_shard = 5;
+    optional string status_shard = 5;
```

Comment:
Everything's optional in proto3, so this is redundant! Saying so explicitly actually causes a warning from `prost`, though that warning gets swallowed by Cargo.

Reply:
Yes, but for scalar types marking it as `optional` causes `prost` [to generate the class](https://github.com/tokio-rs/prost#field-modifiers) with `Option<T>` instead of `T`! If you check the other fields in `ProtoCollectionMetadata`, they are just regular `String`, but `status_shard` is now `Option<String>`.